### PR TITLE
Reproducibility docs: final.final merge into main

### LINKED ISSUE #605: Release the reproducibility docs
This issue tracks merging `reproducibility-docs` into `main` to finish out #487

### PR DESCRIPTION
**Pull request by sjspielman**
I think this one actually closes #605 

In my fork, I synced a couple remaining ewings diffs to match their `main` version, so this PR is from _my fork's_ `reproducibility-docs` into `main`. There are no more ewings diffs!

Let me know if we prefer that I first file `sjspielman/reproducibility-docs` -> `AlexsLemonade/reproducibility-docs` to make sure this repo's branch is its final version, before this PR heading into `main`. I didn't think it mattered too much to do that, but I definitely can!

### CODE REVIEW

**Comment by sjspielman** [comment]
I'm betting the links action failed because GitHub is having some problems right now (although those problems aren't with `Actions` specifically). Locally when I serve, all the links are fine (and spelling is ok too!).

**Comment by allyhawkins** [comment]
> Let me know if we prefer that I first file sjspielman/reproducibility-docs -> AlexsLemonade/reproducibility-docs to make sure this repo's branch is its final version, before this PR heading into main. I didn't think it mattered too much to do that, but I definitely can!

Are we planning on deleting that branch once this goes in? If so then I don't think it matters. But if it's going to stay for any reason then I don't think we want it to be out of date. I can't think of why it would stay though...

**Comment by sjspielman** [comment]
> Are we planning on deleting that branch once this goes in? If so then I don't think it matters

As far as I know we're deleting it, or at least I planned to

**Review by allyhawkins — APPROVED** [review]
LGTM unless others have opinions on the branch situation, I'm good with this.

File: .github/components/dictionary.txt
Comment (jashapiro) [inline .github/components/dictionary.txt:66]: 😢

**Review by jashapiro — APPROVED** [review]
LGTM

**Comment by jashapiro** [comment]
But I'm noting that GH shows the branch as out of date, so it won't allow a merge right now. We should check again after merging in main.

File: .github/components/dictionary.txt
Comment (sjspielman) [reply .github/components/dictionary.txt:66]: oh, @allyhawkins.... ❤️

**Comment by sjspielman** [comment]
Still looks good to me after https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/620/commits/cbf22f346347fc71f208459bc788b92602cf70ff.
The only non-docs changes are still `analyses/cell-type-ewings/.gitignore`, some `template` `yml` files, and the dictionary.

**Comment by sjspielman** [comment]
Woops meant to tag @allyhawkins @jashapiro in case you wanted to look again now that we're up-to-date with `main`! Feel free to give me a thumbs up here.

**Comment by allyhawkins** [comment]
👍

**Comment by sjspielman** [comment]
Oop, another `main` merge needed! Didn't change anything relevant 👍

**Pull request merged**
